en/02_Developer_Guides/03_Forms/How_Tos/01_Encapsulate_Forms.md

```diff
@@ -53,30 +56,33 @@ class PageController extends ContentController
                 NumericField::create('Maximum', 'Maximum')
             )
         );
-        
+
```

Comment:
Change to using `::create` in this whole example pls.

Reply:
That's not in scope. See https://github.com/silverstripe/developer-docs/pull/254#discussion_r1182331298 and the link in the pr description.

We're doing a full pass on all of the docs to just make sure it's not lying, and that's already taking forever. Updating all of the not-best-practice would just make it take even longer, meanwhile there's outright incorrect docs still sitting around. 
We'll update for best practices later on.